hera_mc/mc_session.py

```diff
@@ -728,3 +730,240 @@ def get_paper_temps(self, starttime, stoptime=None):
 
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
+
+    def add_ant_metric(self, obsid, ant, pol, metric, val):
+        """
+        Add a new antenna metric to the M&C database.
+
+        Parameters:
+        ------------
+        obsid: long integer
+            observation identification number
+        ant: integer
+            antenna number
+        pol: string ('x' or 'y')
```

Comment:
Yeah, does that notation exist anywhere else? Right now the data files map to x and y, and the metrics get stored according to that. I can easily map them back to N and E, but I didn't want to do that if everywhere is still x and y.